dev/tools/vitool/pubspec.yaml

```diff
@@ -37,4 +37,4 @@ dev_dependencies:
   term_glyph: 1.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   test_api: 0.2.2 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
 
-# PUBSPEC CHECKSUM: e8ad
+# PUBSPEC CHECKSUM: e9ae
```

Comment:
i recommend making the flutter update-packages --force-upgrade change in a separate PR, so that reverting it is easier if other people have made similar changes in the meantime.

Reply:
Done